### PR TITLE
Adding __bibtex__ and __citation__

### PR DESCRIPTION
**Pull request by bsipocz**
Better late than never (re https://github.com/astropy/photutils/pull/878#issuecomment-504123300)

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -8,6 +8,10 @@ General
    <https://docs.astropy.org/en/stable/wcs/wcsapi.html>`_ is now
    supported. [#899]
 
+ - Added a new ``photutils.__citation__`` and ``photutils.__bibtex__``
+   attributes which give a citation for astroquery in bibtex format. [#926]
```
Comment (larrybradley) [inline CHANGES.rst]:
```suggestion
   attributes which give a citation for photutils in bibtex format. [#926]
```

File: MANIFEST.in [hunk 2]
```diff
@@ -3,7 +3,8 @@ include README.rst
 include CHANGES.rst
 include CODE_OF_CONDUCT.rst
 include CONTRIBUTING.rst
-include CITATION.rst
+include CITATION
+inclide photutils/CITATION
```
Comment (larrybradley) [inline MANIFEST.in]:
```suggestion
include photutils/CITATION
```

**Comment by bsipocz** [comment]
hmm, sorry about the typos 🤦‍♂

**Comment by larrybradley** [comment]
The `CircleCI` failure is real.

**Comment by larrybradley** [comment]
@bsipocz The docs build test failure is real (https://travis-ci.org/astropy/photutils/jobs/570056418#L4115).  The docs page https://github.com/astropy/photutils/blob/master/docs/citation.rst pulls in `CITATION.rst`.  Is there some reason you removed the `.rst` extension from the file?

**Comment by bsipocz** [comment]
Pushed the fix (couldn't check on it locally, my docs build is broken for some reason).

I've removed the extension to be consistent with astropy core, etc.

**Comment by larrybradley** [comment]
Thanks, merging now.  One nice thing about including the .rst extension is that github nicely renders the file, but we can revisit later.

**Comment by bsipocz** [comment]
Oh, good point about the rendering. 
I don't know whether it was a random fluctuation to have a file without extension in astropy or there was a reason behind it.

**Comment by larrybradley** [comment]
I was curious given that everything else (license, contributing, governance, etc.) includes an extension.

**Comment by bsipocz** [comment]
ping @astrofrog as he added it the original file, back in 2013 😉

**Pull request merged**